TrainingTask/Program.cs

```diff
@@ -2,20 +2,23 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
+using System.Threading;
 using System.Threading.Tasks;
-using TrainingTask.Task1;
-using TrainingTask.Task2;
 
 namespace TrainingTask
 {
 	class Program
 	{
 		static void Main(string[] args)
 		{
-			//new Execute().Manage();
+            Matrix matrixInstance = new Matrix();
 
-			new Management().Execute();
-
-		}
+            for (int i = 0; i < 80; i++)
+            {
+                Thread.Sleep(matrixInstance.GenerateThreadSpeed());
```

Comment:
This logic belongs to Matrix class.
Also why do you use Threads, not Tasks?

Reply:
I thought that i can manage threads only in Main. I decided to leave Threads rather than Tasks. Need more practice with Tasks.